go-runtime/schema/analyzers/initialize.go

```diff
@@ -0,0 +1,79 @@
+package analyzers
+
+import (
+	"errors"
+	"go/token"
+	"go/types"
+	"reflect"
+
+	"github.com/TBD54566975/golang-tools/go/analysis"
+	"github.com/TBD54566975/golang-tools/go/analysis/passes/inspect"
+	"github.com/TBD54566975/golang-tools/go/packages"
+)
+
+// Initializer prepares data prior to the schema extractor runs, e.g. loads FTL types for reference by other
+// analyzers.
+var Initializer = &analysis.Analyzer{
+	Name:             "ftlTypeLoader",
+	Doc:              "extracts type aliases to the module schema",
+	Run:              initialize,
+	Requires:         []*analysis.Analyzer{inspect.Analyzer},
+	ResultType:       reflect.TypeFor[initializeResult](),
+	RunDespiteErrors: true,
+}
+
+type initializeResult struct {
+	ftlTypes map[string]*types.Interface
+}
+
+func (r initializeResult) isFtlErrorType(typ types.Type) bool {
+	return r.assertableToFtlType(typ, "builtin", "error")
+}
+
+func (r initializeResult) isFtlContextType(typ types.Type) bool {
+	return r.assertableToFtlType(typ, "context", "Context")
+}
+
+func (r initializeResult) assertableToFtlType(typ types.Type, pkg string, name string) bool {
+	ityp, ok := r.ftlTypes[pkg+"."+name]
+	if !ok {
+		return false
+	}
+	return types.AssertableTo(ityp, typ)
+}
+
+func initialize(pass *analysis.Pass) (interface{}, error) {
+	ctxType, err := loadRef("context", "Context")
+	if err != nil {
+		return nil, err
+	}
+	errType, err := loadRef("builtin", "error")
+	if err != nil {
+		return nil, err
+	}
+
+	return initializeResult{ftlTypes: map[string]*types.Interface{
+		"context.Context": ctxType,
+		"builtin.error":   errType,
+	}}, nil
+}
+
+// Lazy load the compile-time reference from a package.
+func loadRef(pkg, name string) (*types.Interface, error) {
+	pkgs, err := packages.Load(&packages.Config{Fset: token.NewFileSet(), Mode: packages.NeedTypes}, pkg)
+	if err != nil {
+		return nil, err
+	}
+	if len(pkgs) != 1 {
+		return nil, errors.New("expected one package")
```

Comment:
Could we make this `expected one package, got %d`? Since that usually saves some time debugging whenever an error like that does appear